test.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./dist/test');
```

Comment:
ditto

Reply:
Yes maybe. Not sure how to do that on public npm. Want to put up a PR?